<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapsTab</class>
 <widget class="QWidget" name="MapsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0" columnstretch="1,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QScrollBar" name="mapVScrollBar">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollBar" name="mapHScrollBar">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="MapViewer" name="mapViewer" native="true">
     <zorder>mapsView</zorder>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="BDocksZone" name="docks" native="true">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QListView" name="mapsView">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>70</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapViewer</class>
   <extends>QWidget</extends>
   <header location="global">GUI/Tabs/MapTab/mapviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BDocksZone</class>
   <extends>QWidget</extends>
   <header location="global">GUI/Tabs/Docks/bdockszone.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
